documentation/cadl-sample-project/specification/contosowidgetmanager/Contoso.WidgetManager/package.json

```diff
@@ -0,0 +1,11 @@
+{
+  "name": "contoso-widget-manager",
+  "dependencies": {
+    "@cadl-lang/compiler": "latest",
+    "@cadl-lang/rest": "latest",
+    "@cadl-lang/versioning": "latest",
+    "@azure-tools/cadl-autorest": "latest",
+    "@azure-tools/cadl-azure-core": "latest"
```

Comment:
```suggestion
    "@azure-tools/cadl-azure-core": "latest",
    "@azure-tools/cadl-csharp": "latest",
    "@azure-tools/cadl-java": "latest",
    "@azure-tools/cadl-python": "latest",
    "@azure-tools/cadl-typescript": "latest",
```

Reply:
We are still trying to figure out the exact plan for the emitters but I believe currently we don't want these to be in the package.json file in the specs repo but instead be in the language repos like https://github.com/Azure/azure-sdk-for-net/blob/main/eng/csharp-emitter-package.json. This is the approach that @m-nash is working on.

Also I don't believe we want this to be mapped to latest we should pin the versions, at least until things are more stable. @allenjzhang @markcowl do you guys have an opinion one how we should handle this?

On top of those concerns I think we also want to avoid having package.json files in every project but instead I think we want to have one shared one in the specs repo for defining the cadl-autorest emitter versions.

Reply:
@raych1  sdk automation pipeline doesn't support use package.json from SDK language repo now, and if @m-nash 's proposed solution is finalized, we can start to support it.
cc: @weshaggard

Reply:
>@weshaggard However, before the solution is finalized, do you need to add it to package.json in spec repo in this
sample temporilly?

If it is needed to make what we have work currently then yes I'm OK with having it here but if we don't need it yet I'd like to not have it at all from the start. Like I mentioned I'm still not sure how local cadl development will work without a local package.json file here so I'd like some feedback from the cadl team on that from @allenjzhang @markcowl

Reply:
Updated.